Providers/Modules/Plugins/Common/plugin/out_oms_diag.rb

```diff
@@ -0,0 +1,111 @@
+module Fluent
+
+  class OutputOMSDiag < BufferedOutput
+
+    Plugin.register_output('out_oms_diag', self)
```

Comment:
Your PR https://github.com/Microsoft/OMS-Agent-for-Linux/pull/450 also included changes to omsagent.conf to utilize this plugin; do you hope to update omsagent.conf for old OMSAgent versions as well to utilize these files?

Currently, we do not ship omsagent.conf; modules like nxOMSPerfCounter update sections of it on the machine - are you thinking you'll do a similar thing in the future?